okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -326,25 +349,38 @@ class Cookie private constructor(
       this.httpOnly = true
     }
 
+    fun samesite(sameSite: String) = apply {
+      val validSameSites = immutableListOf("lax", "strict", "none")
```

Comment:
Since this type isn't exposed anywhere, there's no need to add additional allocations to ensure it's immutable to external manipulation.
```suggestion
      val validSameSites = listOf("lax", "strict", "none")
```